ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+[{
+  zoneRegex: {
+    en: /^Memoria Misera \(Extreme\)$/,
+  },
+  timelineFile: 'varis-ex.txt',
+  triggers: [
+    {
+      id: 'VarisEx Phase 2',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CCC', capture: false }),
+      run: function(data) {
+        data.phase = 'phase2';
+      },
+    },
+    {
+      id: 'VarisEx Ignis Est',
+      regex: Regexes.startsUsing({ source: 'Ignis Est', id: '4CB6', capture: false }),
+      delaySeconds: 2,
+      response: Responses.getOut(),
+    },
+    {
+      id: 'VarisEx Ventus Est',
+      regex: Regexes.startsUsing({ source: 'Ventus Est', id: '4CC7', capture: false }),
+      delaySeconds: 2,
+      response: Responses.getIn('info'),
+    },
+    {
+      id: 'VarisEx Altius',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CCA', capture: false }),
+      infoText: {
+        en: 'Bait Slashes',
+      },
+    },
+    {
+      id: 'VarisEx Citius',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CF0' }),
+      alertText: function(data, matches) {
+        if (data.me == matches.target) {
+          return {
+            en: 'Buster on YOU',
+          };
+        }
+        if (data.role == 'dps') {
+          return {
+            en: 'Avoid Tanks',
+          };
+        }
+        return {
+          en: 'Buster on ' + data.ShortName(matches.target),
+        };
+      },
+    },
+    {
+      id: 'VarisEx Alea lacta Est',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CD2', capture: false }),
+      response: Responses.getBehind('alarm'),
+    },
+    {
+      id: 'VarisEx Alea lacta Est Front',
+      regex: Regexes.ability({ source: 'Varis Yae Galvus', id: '4CD5', capture: false }),
+      alertText: {
+        en: 'Go Front',
+      },
+    },
+    {
+      id: 'VarisEx Electrified Gunshield',
+      regex: Regexes.ability({ source: 'Varis Yae Galvus', id: '4CD7', capture: false }),
+      delaySeconds: 22,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'VarisEx Reinforced Gunshield',
+      regex: Regexes.ability({ source: 'Varis Yae Galvus', id: '4CD9', capture: false }),
+      condition: (data) => data.phase != 'phase2',
+      delaySeconds: 10,
+      alarmText: {
```

Comment:
Can you change this to `alertText: 'Shields Soon'`.  alarm is very loud, and "stop" on its own is ambiguous (stop moving? stop everything?)

Is there a difference between this and the gunshield below? I'm not sure I understand why there are two triggers.  (Sorry, I haven't run this fight yet, so my knowledge is a bit limited.)

Reply:
I have it set to 2 different attacks as the delay before it goes off is longer in phase 2, unless there is a better way to do it. I feel like 'Shields soon' might be too weak since this attack can easily kill you if you aren't paying attention. (It calls it out about 1-2 seconds before the shields spawn.) Maybe I could make it say 'Stop attacking'.

Reply:
Would this work in this case?
```
durationSeconds: function(data, matches) {
        if (data.phase == 'phase2') {
          return 20
        };
          return 10;
        },

Reply:
"Stop Attacking" or "Pause Attacking" are definitely better.